docker/uwsgi/index.html

```diff
@@ -1,73 +1,93 @@
-    <p>And thanks for coming!</p>
+    <p><a style="color: #a3a3a3" href="https://metamask.io/">Metamask</a> and <a style="color: #a3a3a3" href="https://chrome.google.com/webstore/detail/parity-ethereum-integrati/himekenlppkgeaoeddcliojfddemadig">Parity</a> allow you to access the Ethereum DApps, in both Chrome and Firefox.</p>
+    <h5>2. Get some Kovan Testnet Ether</h5>
+    <p>Paste your ETH address in the <a style="color: #a3a3a3" href="https://gitter.im/kovan-testnet/faucet">Kovan gitter</a> or try <a style="color: #a3a3a3" href="https://github.com/kovan-testnet/faucet">other faucets.</a></p>
+    <h5>3. Enter the demo</h5>
+    <p>Open Metamask on the Ethereum Kovan testnet and test µRaiden.</p>
   </div>
   </div>
-
-
   <div class="columns">
-    <div class="column col-4 col-ml-auto">
-      <div class="empty">
-        <div class="empty-icon">
-          <i class="icon icon-people"></i>
-        </div>
-        <p class="empty-title h5">Buy fortune cookie</p>
-        <p class="empty-subtitle">(words of wisdom, English)</p>
-        <div class="empty-action">
-          <a class="btn btn-primary" id="fortunes_en" href="/fortunes_en">Buy (1 TKN)</a>
-        </div>
+    <div class="column col-5 col-mr-auto col-ml-auto">
+      <div class="empty" style="padding: 32px">
+        <h3>Pay-per-View</h3>
+        <p class="empty-subtitle">Access online content through micropayments</p>
+        <a id="wiki" href="https://wikidemo.micro.raiden.network/wiki/">
+          <img class="grayscale" src="../uwsgi/img/wikiscreen.jpg" width="100%" height="auto">
+        </a>
       </div>
     </div>
-    <div class="column col-4 col-mr-auto">
-      <div class="empty">
-        <div class="empty-icon">
-          <i class="icon icon-emoji"></i>
-        </div>
-        <p class="empty-title h5">Buy fortune cookie</p>
-        <p class="empty-subtitle">(words of wisdom, Chinese)</p>
-        <div class="empty-action">
-          <a class="btn btn-primary" id="fortunes_cn" href="/fortunes_cn">Buy (1 TKN)</a>
-        </div>
-      </div>
-    </div>
-    <div class="column col-4 col-mr-auto">
-      <div class="empty">
-        <div class="empty-icon">
-          <i class="icon icon-bookmark"></i>
-        </div>
-        <p class="empty-title h5">Access Wikipaydia</p>
-        <p class="empty-subtitle">(paywalled proxy for Wikipedia)</p>
-        <div class="empty-action">
-          <a class="btn btn-primary" id="wiki" href="//wikidemo.micro.raiden.network/wiki/">Wiki (1 TKN)</a>
-        </div>
+    <div class="column col-5 col-mr-auto">
+      <div class="empty" style="padding: 32px">
+        <h3>Fortune Cookie Demo
+          <a class="btn btn-primary" id="fortunes_cn" style="margin-top: -5; margin-left: 20px;" href="https://demo.micro.raiden.network/fortunes_cn">中文</a>
+        </h3>
+        <p class="empty-subtitle">Words of wisdom</p>
+        <a id="fortunes_en" href="https://demo.micro.raiden.network/fortunes_en">
```

Comment:
Maybe keeping the links relative could help on testing in this case, but it's not required.

Reply:
Please, use relative path, to ease testing in local host